tests/src/blockchainInfoRetrieverTest.ts

```diff
@@ -76,16 +81,17 @@ describe("BlockchainInfoRetriever.getMinimumFee", async () => {
 	});
 
 	test("error 500, expect ServerError", async () => {
-		nock(fakeUrl)
-			.get(() => true)
-			.reply(500,
-				{
+		const body = {
 					"type": "https://stellar.org/horizon-errors/not_found",
 					"title": "Internal server Error",
 					"status": 500,
 					"detail": "Internal server Error."
-				});
-		await expect(blockchainInfoRetriever.getMinimumFee()).rejects.toEqual(new ServerError(500));
+				};
+		nock(fakeUrl)
+			.get(() => true)
+			.reply(body.status,
+				body);
+		await expect(blockchainInfoRetriever.getMinimumFee()).rejects.toThrowError(new ResourceNotFoundError(body));
```

Comment:
It is mainly semantic because both options are working well. In my opinion it is more clear for the reader to expect 'toThrowError' than 'toEqual' in this case.